cm2350/__init__.py

```diff
@@ -188,6 +185,51 @@ def __init__(self, args=None):
         # Create the MPC5674 emulator with the default configuration values
         self.emu = MPC5674_Emulator(defconfig=self.defconfig, docconfig=self.docconfig, args=args)
 
+        if self.emu.custom_entrypoint is not None:
+            # If a custom entry point was set, there are a few emulator settings 
+            # that need to be updated to ensure that the application can operate 
+            # correctly., the MSR must be updated to enable machine check 
+            # exceptions, and the PPC timebase should be enabled.
+            self.emu.setRegister(ppc_regs.REG_MSR, 0x1000)
+            self.emu.setRegister(ppc_regs.REG_HID0, 0x4000)
```

Comment:
should these magic numbers be constants like 
```
MSR_MACHINECHECK
HID0_TIMEBASE
```
 or something?

Reply:
yes they should be

Reply:
fixing